### PR TITLE
feat: Added stories UI

### PR DESCRIPTION
**Pull request by mlchct**
## Summary:
Added scrollable stories

## Changes:
* Refractored Home Page collectionView
* Modified post cells to make one post cell as one single view
* Made header view as UICollectionReusableView
* Implemented stories and posts as 2 different sections with horizontal and vertical scrolls

## Screenshots:

https://github.com/Danila-Bernat-iOS/instagram-like-trainee/assets/165797844/e9ba0c0a-fd72-47b5-8c6a-a6f13c4acf69

### CODE REVIEW

File: Instagram-like-trainee/Resources/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -9,8 +9,8 @@
 "likesLabel" = "Likes: 143 569";
 "commentLabel" = "johnnydepp 🔥🔥🔥";
 "userNameLabel" = "elon_official";
-"homeFeedHeaderCellID" = "HomeFeedHeaderCell";
-"postActionCellID" = "postActionCell";
-"postCaptionCellID" = "PostCaptionCell";
+"postCellID" = "postCell";
+"addStoryCollectionViewCellID" = "addStoryCollectionViewCell";
+"storiesCollectionViewCellID" = "storiesCollectionViewCell";
```
[the remark this answers is not in the document]
Comment (mlchct) [reply Instagram-like-trainee/Resources/en.lproj/Localizable.strings:14]: Done

File: Instagram-like-trainee/Screens/HomePage/View/CollectionViewCells/StoriesCells/AddStoryCollectionViewCell.swift [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func configureUI() {
+        contentView.backgroundColor = .systemBackground
+        configureProfileImageView()
+        configurePlusSignImageView()
+    }
```
[the remark this answers is not in the document]
Comment (mlchct) [reply Instagram-like-trainee/Screens/HomePage/View/CollectionViewCells/StoriesCells/AddStoryCollectionViewCell.swift:48]: Done

File: Instagram-like-trainee/Screens/HomePage/View/CollectionViewCells/StoriesCells/StoriesCollectionViewCell.swift [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+final class StoriesCollectionViewCell: UICollectionViewCell {
+    private var imageView: UIImageView = {
+        let imageView = UIImageView(image: UIImage(systemName: "questionmark.circle"))
+        imageView.contentMode = .scaleAspectFill
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        return imageView
+    }()
+
+    private var gradientView: UIView = {
+        let view = UIView(frame: .zero)
```
[the remark this answers is not in the document]
Comment (mlchct) [reply Instagram-like-trainee/Screens/HomePage/View/CollectionViewCells/StoriesCells/StoriesCollectionViewCell.swift:19]: Done

File: Instagram-like-trainee/Screens/HomePage/View/CollectionViewCells/StoriesCells/AddStoryCollectionViewCell.swift [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+//  TableViewTest
+//
+//  Created by Mikhail Kalatsei on 09/04/2024.
+//
+
+import UIKit
+
+final class AddStoryCollectionViewCell: UICollectionViewCell {
+    private let profileImageView: UIImageView = {
+        let imageView = UIImageView(image: UIImage(systemName: "questionmark.circle"))
```
[the remark this answers is not in the document]
Comment (mlchct) [reply Instagram-like-trainee/Screens/HomePage/View/CollectionViewCells/StoriesCells/AddStoryCollectionViewCell.swift:12]: Done

**Review by e-garbuzov-wise — APPROVED** [review]

File: Instagram-like-trainee/Screens/HomePage/View/CollectionViewCells/PostCell/PostFooterView.swift [hunk 5]
```diff
@@ -46,48 +44,68 @@ final class PostActionCell: UICollectionViewCell {
...
 
     private func configureUI() {
-        addSubview(likeButtonImageView)
-        addSubview(commentButtonImageView)
-        addSubview(shareButtonImageView)
-        addSubview(addFavButtonImageView)
+        configureLikeButtonImageView()
+        configureCommentButtonImageView()
+        configureShareButtonImageView()
+        configureAddFavButtonImageView()
```
[the remark this answers is not in the document]
Comment (mlchct) [reply Instagram-like-trainee/Screens/HomePage/View/CollectionViewCells/PostCell/PostFooterView.swift:61]: Done

**Pull request merged**